src/Method/Prototype.php

```diff
@@ -140,29 +111,22 @@ public function getParameterObjects()
      * Retrieve a single parameter by name or index
      *
      * @param  string|int $index
-     * @return null|\Laminas\Server\Method\Parameter
+     * @return null|Parameter
```

Comment:
no additional value in this return docblock, can be removed.

Reply:
Removed.